packages/paste-website/src/pages/components/timeline/index.mdx

```diff
@@ -0,0 +1,334 @@
+  TimelineComposition,
+  TimelineGrouped,
+  TimelineIcon,
+  TimelineStart,
+  TimelineTimestamp,
+} from "../../../component-examples/TimelineExamples.ts";
+import { ResponsiveImage } from '../../../components/ResponsiveImage';
+
+import { SupportIcon } from "@twilio-paste/icons/esm/SupportIcon";
+import Changelog from "@twilio-paste/timeline/CHANGELOG.md";
+import packageJson from "@twilio-paste/timeline/package.json";
+import InfiniteTimeline from '../../../assets/images/timeline-infinite.png';
+import { SidebarCategoryRoutes } from "../../../constants";
+import ComponentPageLayout from "../../../layouts/ComponentPageLayout";
+import { getFeature, getNavigationData } from "../../../utils/api";
+export default ComponentPageLayout;
+
+export const getStaticProps = async () => {
+  const navigationData = await getNavigationData();
+  const feature = await getFeature("Timeline");
+  return {
+    props: {
+      data: {
+        ...packageJson,
+        ...feature,
+      },
+      navigationData,
+      mdxHeadings,
+      pageHeaderData: {
+        categoryRoute: SidebarCategoryRoutes.COMPONENTS,
+        githubUrl: "https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/timeline",
+        storybookUrl: "/?path=/story/components-timeline--default-timeline",
+      },
+    },
+  };
+};
+
+<LivePreview scope={{ Timeline, TimelineItem }} noInline>
+  {TimelineStart}
+</LivePreview>
+
+## Guidelines
+
+### About Timeline
+
+A Timeline can be used to display historical, current or future events or information in chronological order based on when they happened or are scheduled to happen.
+
+Timelines can be used in different use cases, such as tracking milestones, monitoring system logs, or visualizing user activity.
+
+### Accessibility
+
+<UnorderedList>
+  <ListItem>
+    Timeline is an ordered list (&lt;ol&gt;) and TimelineItem is a list item (&lt;li&gt;).
+  </ListItem>
+
+  <ListItem>
+    The only tabbable and interactive part of a Timeline is the toggle button when using the <inlineCode>collapsible</inlineCode> prop
+    <UnorderedList marginBottom="space0">
+      <ListItem>
+        <inlineCode>collapsible</inlineCode> TimelineItems use the <Anchor href="/components/summary-detail">Summary Detail</Anchor> component. The toggle button is labeled by the value of the <inlineCode>timestamp</inlineCode> prop.
+        If the value is too verbose for a screen reader or not a clear label for the collapsed content, provide a helpful label for the button using the <inlineCode>collapsibleLabelText</inlineCode> prop on TimelineItem
+      </ListItem>
+    </UnorderedList>
+  </ListItem>
+  <ListItem>
+    Don’t use icons TimelineItem markers to represent status like warning, error, or success. Since the marker is non-interactive, relying on icons for semantic meaning can cause accessibility challenges, particularly for people using screen readers.
+  </ListItem>
+</UnorderedList>
+
+### Timeline vs. Progress Steps
```

Comment:
suggestion: can we add this section to progress steps docs too? in either PR

Reply:
will do in progress-steps pr